src/util/Constants.ts

```diff
@@ -0,0 +1 @@
+export { version } from "../../package.json"
```

Comment:
Path is too long so use baseUrl option in tsconfig
```suggestion
export { version } from "package.json"
```